### PR TITLE
Deleted Zscalar web security ad IIS-10 app docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/security-threat-detection/zscaler-internet-access.md [hunk 1]
```diff
@@ -174,45 +174,39 @@ _sourceCategory=ZIA
 | top 10 baseurl by _count
 ```
 
+## Install the Zscaler Internet Access app
 
-
-## Install the Zscaler Internet Access App
-
-This section provides instructions on how to install the Zscaler Internet Access App, and provides examples of each of the dashboards. The App preconfigured searches and [Dashboards](/docs/integrations/security-threat-detection/Zscaler-Web-Security#Dashboards) provide easy-to-access visual insights into your data.
+This section provides instructions on how to install the Zscaler Internet Access app, and provides examples of each of the dashboards. The preconfigured searches and dashboards provides you an easy-to-access visual insights into your data.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/zscaler-internet-access.md:179]:
```suggestion
This section describes how to install the Zscaler Internet Access app, and provides examples of each of the dashboards. The preconfigured searches and dashboards provides you an easy-to-access visual insights into your data.
```

**Review by kimsauce — APPROVED** [review]

**Review by kimsauce — CHANGES_REQUESTED** [review]
You're still missing the old-to-new redirects. See https://help.sumologic.com/docs/contributing/remove-doc/#step-1-create-a-301-redirect.

**Comment by JV0812** [comment]
Thanks Kim, I have updated this accordingly.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> Thanks Kim, I have updated this accordingly.

Awesome, thank you - you are fast 🚗

**Pull request merged**